rocky/reports/views/base.py

```diff
@@ -79,6 +79,7 @@ def get_context_data(self, **kwargs):
 class BaseReportView(OOIFilterView):
     def setup(self, request, *args, **kwargs):
         super().setup(request, *args, **kwargs)
+        self.valid_time = self.observed_at
```

Comment:
There is no need to have two attributes on the view instance that do the same. The cached property can just be named `valid_time` and then we can use `self.valid_time` everywhere.